core/lib/spree/core/validators/email.rb

```diff
@@ -1,30 +0,0 @@
-# frozen_string_literal: true
-
-module Spree
-  # == An ActiveModel Email Validator
-  #
-  # === Usage
-  #
-  #     require 'spree/core/validators/email'
-  #
-  #     class Person < ApplicationRecord
-  #       validates :email_address, 'spree/email' => true
-  #     end
-  #
-  class EmailValidator < ActiveModel::EachValidator
-    EMAIL_REGEXP = /\A([^@\.]|[^@\.]([^@\s]*)[^@\.])@([^@\s]+\.)+[^@\s]+\z/
-
-    def validate_each(record, attribute, value)
-      unless value =~ EMAIL_REGEXP
-        record.errors.add(attribute, :invalid, { value: value }.merge!(options))
-      end
-    end
-  end
-end
-
-# @private
-EmailValidator = ActiveSupport::Deprecation::DeprecatedConstantProxy.new(
```

Comment:
This is the only part to remove if we want to get rid of the deprecation message.

Reply:
Yeah makes sense I didn't notice that it was a proxy 👍